lib/vast/Conversion/FromHL/EmitLazyRegions.cpp

```diff
@@ -133,6 +133,7 @@ namespace vast
         static conversion_target create_conversion_target(mcontext_t &context) {
             conversion_target target(context);
             target.addLegalDialect< vast::core::CoreDialect >();
+            target.markUnknownOpDynamicallyLegal([](auto){ return true; });
```

Comment:
why is this needed now?

Reply:
This should really always be present - otherwise if there is an op that is not explicitly specified by other callback (or is from some "external" dialect to the pass), it will be treated as illegal and conversion will fail even if no work was to be done for that op.